src/server/pfs/drive/driver.go

```diff
@@ -773,6 +913,21 @@ func addDirs(diffInfo *pfs.DiffInfo, child *pfs.File) {
 	}
 }
 
+func removeFromDir(diffInfo *pfs.DiffInfo, child *pfs.File) {
```

Comment:
You use the word "remove" in a few places in this file, could we change those to "delete?" In my mind their synonyms so I'd prefer to stick to just one of them.